_data/sites.json

```diff
@@ -2551,6 +2551,16 @@
         ]
     },
 
+    {
+        "name": "Garena",
+        "url": "https://www.garena.sg/",
```

Comment:
Let's make this the support link instead: https://www.garena.sg/support/

Reply:
Done!